cli/azd/pkg/ext/models.go

```diff
@@ -0,0 +1,186 @@
+package ext
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/osutil"
+)
+
+type ScriptType string
+type ScriptLocation string
+
+const (
+	ScriptTypeBash        ScriptType     = "bash"
```

Comment:
Should this be `sh` and not `bash` given that we moved to writing a `#!/bin/sh` shebang for inline "bash" scripts?